ui/src/features/Editor/components/ParamsPanel/components/ParamEditor/index.tsx

```diff
@@ -79,6 +81,12 @@ const ParamEditor: React.FC<Props> = ({
   const handleUpdate = () => {
     if (activeTab === "params") {
       onUpdate(nodeId, updatedParams, "params");
+    } else if (nodeType === "subworkflow" && nodeMeta.subworkflowId) {
+      onUpdate(nodeId, updatedCustomization, "customizations");
+      onWorkflowRename?.(
+        nodeMeta?.subworkflowId,
+        updatedCustomization?.customName || nodeMeta?.officialName,
+      );
```

Comment:
Need to lint this probably.
```suggestion
    } else if (nodeType === "subworkflow" && nodeMeta.subworkflowId) {
      onUpdate(nodeId, updatedCustomization, "customizations");
      if (updatedCustomization?.customName) {
        onWorkflowRename?.(
          nodeMeta?.subworkflowId,
          updatedCustomization.customName,
        );
      }
```

Reply:
I think for this the original is probably ok as if the user has not added a customName and has left it blank we would want the official name to be used (for other nodes this is the case).